shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -1,7 +1,7 @@
 <template name="loginButtons">
   {{!-- Data context is an instance of AccountsUi. --}}
   {{#if isDemoUser}}
-    <a class="{{#if expiringSoon}}expiring-soon{{/if}}">Demo {{demoTimeLeft}} ▾</a>
+    <a>Create Account ▾</a>
```

Comment:
s/Create Account/Sign up for free/ ? But I can live with this as-is too. (I defer having a strong preference about the wording of this to @neynah .)